website/docs/r/iam_trusted_profiles.html.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_iam_trusted_profile"
+description: |-
+  Manages iam_trusted_profile.
+subcategory: "IAM Identity Services"
+---
+
+# ibm_iam_trusted_profile
+
+Provides a resource for iam_trusted_profile. This allows iam_trusted_profile to be created, updated and deleted.
```

Comment:
The description should be edited something like this:

Create, update and delete an IVM trusted profile resource. For more information, about trusted profile, see [Create  a trusted profile](https://cloud.ibm.com/apidocs/iam-identity-token-api#create-link)